### ISSUE TITLE
IdsExample: Accessibility WCAG Guideline:4.1.2 Name, Role, Value: Name of 'show/ hide' button is not defined.

### ISSUE THREAD

**Issue by CindyMercadoReyes** [infor-design/enterprise-wc#1386]
Describe the bug : Name of 'show/ hide' button is not defined.

To Reproduce
"1. Launch the URL: https://main.wc.design.infor.com/ids-input/example.html
2. Navigate to 'show/ hide' button using arrow keys and verify"
3. Name of 'show/ hide' button is not defined. Screen reader is reading it as button button.

Expected behavior :Name of 'show/ hide' button should be properly defined.

User Impact:  If name will not be defined then screen reader user will not be able to understand the purpose of button.

Screenshots If applicable, add screenshots to help explain your problem.: https://app.screencast.com/1Bb6lgckeQgDw

Dev Recommendation:  
"[Provide proper aria label to the button.](http://www.w3.org/TR/WCAG20-TECHS/ARIA6.html)
[http://www.w3.org/TR/WCAG20-TECHS/ARIA6.htm](http://www.w3.org/TR/WCAG20-TECHS/ARIA6.html)l"

Platform
    Windows 10/Chrome/NVDA

Additional context
NA
